src/scss/includes/panels/poi_panel.scss

```diff
@@ -25,8 +25,8 @@ $HEADER_SIZE: 40px;
 }
 
 .poi_panel__image {
-  width: 100px;
-  height: 100px;
+  width: 90px;
+  height: 90px;
```

Comment:
do we really have to reduce the image size, that is already quite small ?

Reply:
:man_shrugging: I don't even see the cases where it's used, so I don't have any opinion. Maybe we can simply revert this change for now to merge, and discuss every UI tweaks in a following PR.

Reply:
Thanks for the IRL explanation, so this is the illustration picture (most often photo) that exists for some POIs. I agree this is important, so I reverted it at 100px for now. https://github.com/QwantResearch/erdapfel/pull/289/commits/90dc60f06d4fb05814d8fdec7afe623130384145
I'll probably change the layout of this component later so it doesn't depend on fixed height, so we don't have a big empty space when this photo doesn't exist.